packages/@react-spectrum/s2/src/Radio.tsx

```diff
@@ -28,7 +28,7 @@ export interface RadioProps extends Omit<AriaRadioProps, 'className' | 'style' |
   /**
    * The label for the element.
    */
-  children?: ReactNode
```

Comment:
I think this one and some others where the children are the label can be optional since they could be aria-label'ed instead

Reply:
Thanks, I noticed Checkbox was like this, but missed this. I'll double check the rest.